xil_cloud/docker-compose.yml

```diff
@@ -0,0 +1,139 @@
+version: '3'
+services:
+
+  carlasim:
```

Comment:
Should probably name this container carma-simulation or cda-sim instead of carla-sim since there's more than just CARLA running in there.